client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-homepage.tsx

```diff
@@ -56,12 +54,7 @@ const ScreenHomepage = ( {
 				/>
 			</div>
 			<div className="screen-container__footer">
-				<NavigatorBackButton
-					as={ Button }
-					className="pattern-assembler__button"
-					onClick={ onDoneClick }
-					primary
-				>
+				<NavigatorBackButton as={ Button } className="pattern-assembler__button" primary>
```

Comment:
No, since I assume it's only fired when you're seeing the pattern list on the screen and click on that "Done" button. Does it make sense? We can add a new event to this button later.

![image](https://user-images.githubusercontent.com/13596067/223393052-9652eba8-3f9a-4a4d-bc64-3311b8df4295.png)